### DISCUSSION TITLE
关于音乐播放的问题，

### DISCUSSION THREAD

**Discussion by liuluoqianqiu** [theme-shoka-x/hexo-theme-shokaX#190] [Q&A]
### 在开始之前......

- [X] 我理解 Issue 是用于反馈和解决问题的，将尽可能提供更多信息帮助问题解决
- [X] 我填写了简短且清晰明确的标题，以便开发者在收到 Issue 时能快速确定大致问题。而不是"一个建议"、"加功能"等
- [X] 我确认最新版尚未添加此功能，且此前没有人提出过此问题
- [X] 为保证 Issue 及时得到回复，我已选择至少一名不在 busy 状态的组织成员作为`assignees`

### 建议描述

看player.ts里面的写法，好像只能支持网易云和qq音乐的链接，但实际上我想用自己本地的音乐传到网盘做成在线播放的链接形式，这样我就可以把博客做成一个类似于英语听力的网站，那么这个地方应该怎么实现呢，附上测试链接，https://audio.com/freewill/audio/lots-of-water-003

### 注意事项

- [X] 我已确认以上事项

**Comment by FidStyle** [comment]
考虑[hexo-tag-mmedia](https://github.com/u2sb/hexo-tag-mmedia)提供的[meting](https://www.u2sb.com/OpenSw/hexo-tag-mmedia/meting.html)以及[Aplayer](https://www.u2sb.com/OpenSw/hexo-tag-mmedia/aplayer.html)，应该可以自定义播放源

@liuluoqianqiu
